Automatically change sex when replacing characters in studio

ManlyMarco opened the pull request.

ManlyMarco committed:
Automatically change sex when replacing characters in studio

im-mi commented:
Hey. Sorry it took a while to get back to you.

This is a pretty good idea, although the current implementation seems to have a few issues. Here are the one's I noticed:
* Replacing a character creates an extra copy of the dropped character.
* The replacing character doesn't get adjusted to account for sex-based limitations (height, clothes, etc.). This lasts until the scene gets saved and loaded (though I'll admit the anti-gravity bras are pretty cool while they last).

ManlyMarco committed:
Fixed character being added twice in studio when replacing

ManlyMarco committed:
Added Shift key modifier - if held scenes will be imported and characters will always be created new

ManlyMarco committed:
Set file version to version number; Expose GUID field; Version up to v1.3

ManlyMarco committed:
Update README.md

ManlyMarco commented:
Fixed the `Replacing a character creates an extra copy of the dropped character.` issue, not sure how to approach the other problem.

I also added ability to hold shift to import scenes and to force add characters instead of replacing selected.

ManlyMarco committed:
Fixed compatibility with Darkness DLC; Version up to 1.3.1

ManlyMarco committed:
Merge branch 'master' of https://github.com/ManlyMarco/KoikatuDragAndDrop

ManlyMarco committed:
Fix compatibility with party cards

ManlyMarco committed:
Merged version for AIS by @YumeMienai in PR #1 and restructured project to extract shared code

ManlyMarco committed:
Update README.md
